### ISSUE TITLE
onnx model has no weights and bias

### ISSUE THREAD

**Issue by ArtyZe** [jinfagang/DCNv2_latest#5]
Hello, thanks for your great work!
Now I can export onnx model with dcn op successfully. But my onnx model is not correct:
![image](https://user-images.githubusercontent.com/32027268/89510742-8cfb4380-d803-11ea-8321-7a60eef26f91.png)
As my weights and bias have not been export to onnx.

Have you ever met same error? Thanks!

**Comment by jinfagang** [comment]
@ArtyZe DCN's onnx is a custom op not listed on ONNX standard, your visualization result might not accurate. For validation, you can check your onnx inference result using onnx-runtime.

**Comment by ArtyZe** [comment]
> @ArtyZe How did you manage to export to ONNX model? When I try to export to onnx, I get `RuntimeError: No Op registered for Plugin with domain_version of 9`
> Thanks in advance for you help!

Hi, You got see code below in file dcnv2_onnx.py ?
![image](https://user-images.githubusercontent.com/32027268/96683952-6b930800-13ad-11eb-8f23-b78738e8231b.png)
It's a torch supported way to register ops to torch/onnx.
So 1. replace your dcnv2.py file with dcnv2_onnx.py and then then export model to onnx 2. add above code to your own code and then export model to onnx

**Comment by ArtyZe** [comment]
> @ArtyZe DCN's onnx is a custom op not listed on ONNX standard, your visualization result might not accurate. For validation, you can check your onnx inference result using onnx-runtime.

You are right. It's a tool bug of netron for custom ops. Data exist but netron displays nothing. It's right when I export data of this op with python

**Comment by austinmw** [comment]
@maxclaey did you find a solution to this? I'm running into the same issue. @jinfagang  Do you happen to know what this error comes from?
